docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -0,0 +1,75 @@
+
+
+Overview
+------------------------
+
+Beneficial owners can exercise ownership-or-control *directly* in a company (entity) or *indirectly*, via intermediary entities and arrangements. It must be clear in a BODS dataset which people are declared as beneficial owners of which companies (and which companies are intermediaries). And it must be clear what control or ownership beneficial owners ultimately have (whether it is direct or indirect).
+
+.. figure:: ../../_assets/RepresentingChainsBODS-RealWorld.svg
+   :alt: Person 1 indirectly holds a 15 percent shareholding in Company E, via an intermediary: Company A. Person 2 directly holds 32 precent of Company E's shares.
+   :width: 500px
+   :align: center
+
+In the above situation, Person 1 *indirectly* benefits from a 15% shareholding in Company E and Person 2 has a *direct* 32% shareholding in Company E. In a jurisdiction where people with a shareholding (direct or indirect) of over 10% in a company should be disclosed as beneficial owners, both Person 1 and Person 2’s interests would be declared. Additionally, the jurisdiction may require that details of Person 1’s indirect interest are disclosed. That is: that some details of the chain Company E - Company A - Person 1 are also disclosed. 
+
+In BODS, the following properties are used to represent such information disclosure:
+
+* ``beneficialOwnershipOrControl`` (See :ref:`schema-interest`)
+* ``directOrIndirect`` (See :ref:`schema-interest`)
+* ``componentStatementIDs`` (See :ref:`schema-ownership-or-control-statement`)
+* ``isComponent`` (See :ref:`schema-entity-statement`)
+
+Requirements
+------------------------
+
+Representing beneficial ownership
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If a person is a beneficial owner of an entity, E, whether directly or indirectly, and one of them is required to declare this beneficial ownership:
+
+1. There MUST be a *primary* Ownership-or-control Statement connecting the two which represents the beneficial ownership relationship. In particular: 
+
+   a. the entity’s Statement MUST be the ``subject``;
+   b. the person’s Statement MUST be the ``interestedParty``;
+   c. ``isComponent`` MUST be false;
+   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included in this primary Ownership-or-Control Statement if known; and
+   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'.
+
+2. If beneficial ownership is known to be exercised indirectly, via intermediary entities or arrangements then the ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
```

Comment:
Depending on what you think about rewriting the phrase "entities and arrangements" I'll note that it appears a couple of times here as well.

Reply:
I've removed '...and arrangements' in each case